tests/test_wave_03.py

```diff
@@ -127,12 +125,10 @@ def test_mark_complete_missing_task(client):
 
     # Assert
     assert response.status_code == 404
-
-    # raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
-
+    
+    # Added assertion
+    assert "'message': 'Task #1 was not found' in response_body"
```

Comment:
This assertion has no comparison since there's no `==` in this line of code. You have the assert keyword and then a string. Since the string is not empty, it will evaluate to a Truthy value so the test will pass, but you're not actually testing anything. 

This assertion could be written like
```python
assert response_body == {"message": "Task #1 was not found' in response_body"}
```